startup/GafferScene/arnoldOptions.py

```diff
@@ -0,0 +1,237 @@
+Gaffer.Metadata.registerValue( "option:ai:GI_transmission_samples", "defaultValue", IECore.IntData( 2 ) )
+Gaffer.Metadata.registerValue(
+	"option:ai:GI_transmission_samples",
+	"description",
+	"""
+	Controls the number of rays traced when
+	computing specular refractions.
+	"""
+)
+
+Gaffer.Metadata.registerValue( "option:ai:GI_sss_samples", "label", "SSS Samples" )
+Gaffer.Metadata.registerValue( "option:ai:GI_sss_samples", "defaultValue", IECore.IntData( 2 ) )
+Gaffer.Metadata.registerValue(
+	"option:ai:GI_sss_samples",
+	"description",
+	"""
+	Controls the number of rays traced when
+	computing subsurface scattering.
+	"""
+)
+
+Gaffer.Metadata.registerValue( "option:ai:GI_volume_samples", "label", "Volume Samples" )
+Gaffer.Metadata.registerValue( "option:ai:GI_volume_samples", "defaultValue", IECore.IntData( 2 ) )
+Gaffer.Metadata.registerValue(
+	"option:ai:GI_volume_samples",
+	"description",
+	"""
+	Controls the number of rays traced when
+	computing indirect lighting for volumes.
+	"""
+)
+
+Gaffer.Metadata.registerValue( "option:ai:light_samples", "label", "Light Samples" )
+Gaffer.Metadata.registerValue( "option:ai:light_samples", "defaultValue", IECore.IntData( 0 ) )
+Gaffer.Metadata.registerValue(
+	"option:ai:light_samples",
+	"description",
+	"""
+	Specifies a fixed number of light samples to be taken at each
+	shading point. This enables "Global Light Sampling", which provides
+	significantly improved performance for scenes containing large numbers
+	of lights. In this mode, the `samples` setting on each light is ignored,
+	and instead the fixed number of samples is distributed among all the
+	lights according to their contribution at the shading point.
+
+	A value of `0` disables Global Light Sampling, reverting to the original
+	per-light sampling algorithm.
+	"""
+)
+
+Gaffer.Metadata.registerValue( "option:ai:GI_total_depth", "label", "Total Depth" )
+Gaffer.Metadata.registerValue( "option:ai:GI_total_depth", "defaultValue", IECore.IntData( 10 ) )
+Gaffer.Metadata.registerValue(
+	"option:ai:GI_total_depth",
+	"description",
+	"""
+	The maximum depth of any ray (Diffuse + Specular +
+	Transmission + Volume).
+	"""
+)
+
+Gaffer.Metadata.registerValue( "option:ai:GI_diffuse_depth", "label", "Diffuse Depth" )
+Gaffer.Metadata.registerValue( "option:ai:GI_diffuse_depth", "defaultValue", IECore.IntData( 2 ) )
+Gaffer.Metadata.registerValue(
+	"option:ai:GI_diffuse_depth",
+	"description",
+	"""
+	Controls the number of ray bounces when
+	computing indirect illumination ("bounce light").
+	"""
+)
+
+
+Gaffer.Metadata.registerValue( "option:ai:GI_specular_depth", "label", "Specular Depth" )
+Gaffer.Metadata.registerValue( "option:ai:GI_specular_depth", "defaultValue", IECore.IntData( 2 ) )
+Gaffer.Metadata.registerValue( "option:ai:GI_specular_depth", "description", "Controls the number of ray bounces when computing specular reflections." )
```

Comment:
Seems to be a duplicate of the one below.

Reply:
Well spotted! 588447e